setup.cfg

```diff
@@ -6,5 +6,71 @@ known_djangoapp = model_utils
 known_first_party = fonzie
 sections = FUTURE,STDLIB,THIRDPARTY,DJANGO,DJANGOAPP,EDX,FIRSTPARTY,LOCALFOLDER
 
+[metadata]
+name = fonzie
+version = 0.1.0
+description = A FUN API for Open edX
+long_description = file: README.rst
+author = Open FUN (France Universite Numerique)
+author_email = julien.maupetit@fun-mooc.fr
```

Comment:
Do we have a generic contact email such as: `opensource@fun-mooc.fr` or `hello@openfun.fr`?

Reply:
Yes we have `fun.dev@fun-mooc.fr` for that.

Reply:
Maybe it would be a job for openfun.fr but we nothing yet to handle emails on this domain :-( and all our email boxes are in fun-mooc.fr...